install/linux/usr/share/odemis/hwtest/zeiss-only.odm.yaml

```diff
@@ -0,0 +1,127 @@
+        external: "SEM E-beam",
+        internal: "EBeam control"
+    },
+#    children: {
+#        "detector": "SEM Detector full",
+#    },
+    init: {},
+    properties: {
+        scale: [8, 8], # (ratio) : start with a pretty fast scan
+        dwellTime: 10.e-6, # s
+    },
+    affects: ["SEM Detector"]
+}
+
+# To use the "external" control from the API instead of the analog
+# Merges the analog and digital detectors (zeiss one and the one from the DAQ board), to provide an "external" signal via the API
+#"SEM Detector full": {
+#    role: se-detector,
+#    dependencies: {
+#        external: "SEM Detector",
+#    },
+#    init: {},
+#    affects: ["SEM Detector", "VIS-NIR Camera"],
+#}
+
+"SEM Scan Interface": {
+    class: semcomedi.SEMComedi,
+    role: null,
+    init: {device: "/dev/comedi0"},
+    # more detectors can be added, if necessary
+    children: {
+        scanner: "SEM E-beam",
+        detector0: "SEM Detector",
+    }
+}
+
+# Connect:
+# X -> AO 0
+# Y -> AO 1
+# Ground -> AO GND
+"SEM E-beam": {
+    role: null,
+    init: {
+        channels: [0, 1],
+        # On Delmic scanning box v2, the voltage is x2, so need to specify twice smaller values than needed.
+        # but could be due to 4:3 image. For now Odemis only handles 1:1 images.
+        max_res: [4096, 3072], # px, to force 4:3 ratio
+        limits: [[-4.85, 4.85], [-3.64, 3.64]],  # V
+        park: [-5, -5], # V was -4.3
+        # Digital output port mapping on the Delmic scanning box v2:
+        # 0 = Relay
+        # 1 = Open drain output (Y0.0)
+        # 2 = Digital Out 1
+        # 3 = Digital Out 0
+        # 4 = Status led
+        # output ports -> True (indicate scanning) or False (indicate parked)
+        # Note: for now the "external" signal is not working (because it needs 5V?)
+        scanning_ttl: {
+            3: [True, "external"],
+            4: True
+        },
+        settle_time: 120.e-6, # s
+        hfw_nomag: 0.2756, # m  was 0.098
+    },
+    properties: {
+        scale: [8, 8], # (ratio) : start with a pretty fast scan
+        dwellTime: 2.e-6, # s
+        magnification: 100, # (ratio)
+    },
+}
+
+# Must be connected on AI1/AI9 (differential)
+"SEM Detector": { # aka ETD
+    role: se-detector,
+    init: {
+        channel: 1,
+        limits: [0, 5], # V
+    },
+}
+
+"SEM via API": {
+    class: zeiss.SEM,
+    role: null,
+    init: {
+        port: "/dev/ttySEM*",
+        # eol: "\r",  # If using a Point Electronic SEM, use this line
+    },
+    children: {
+        scanner: "EBeam control",
+        focus: "EBeam Focus",
+        stage: "Sample Stage",
+    }
+}
+
+"EBeam control": {
+    role: null,
+    init: {
+        hfw_nomag: 0.127,  # m, theoretical HFW if mag == 1x, TODO: depends on the voltages
+    },
+}
+
+"EBeam Focus": {
+    role: ebeam-focus,
+}
+
+"Sample Stage": {
+    role: stage,
+    affects: ["SEM E-beam full"],
+    init: {
+        inverted: ["x", "y"], # TODO: z ??
```

Comment:
Does this todo still need to be in?

Reply:
I guess with the METEOR work, we should now know whether the Z goes up or down. Normally the convention in Odemis is that an increase in Z causes the sample to be closer to the e-beam source. 
Do you know?

Reply:
Good question we have to check with Jochem and Karisma for that.

Reply:
z is positive upwards I just heard from Karishma so I guess we can leave inversion of z out.